cpp/ql/test/library-tests/dataflow/dataflow-tests/flowOut.cpp

```diff
@@ -1,20 +1,74 @@
-int source();
-void sink(int);
+int source(); char source(bool);
+void sink(int); void sink(char);
 
 void source_ref(int *toTaint) { // $ ir-def=*toTaint ast-def=toTaint
     *toTaint = source();
 }
-
-
-
+void source_ref(char *toTaint) { // $ ir-def=*toTaint ast-def=toTaint
+    *toTaint = source();
+}
 void modify_copy(int* ptr) { // $ ast-def=ptr
   int deref = *ptr;
   int* other = &deref;
   source_ref(other);
 }
 
-void test_output() {
+void test_output_copy() {
    int x = 0;
    modify_copy(&x);
-   sink(x); // $ SPURIOUS: ir
+   sink(x); // clean
+}
+
+void modify(int* ptr) { // $ ast-def=ptr
+  int* deref = ptr;
+  int* other = &*deref;
+  source_ref(other);
+}
+
+void test_output() {
+   int x = 0;
+   modify(&x);
+   sink(x); // $ SPURIOUS: ir MISSING: ast
```

Comment:
There's supposed to be flow here, right? I.e. it shouldn't have a `SPURIOUS` label.

Reply:
Oops, you're right. That was a copy/paste fail 😅

Reply:
Fixed in 5d9973bb57d99ac8fd80a64d11260a7219d26942.